appium/webdriver/webdriver.py

```diff
@@ -160,7 +160,6 @@ def __init__(
             self._addCommands()
 
         self.error_handler = MobileErrorHandler()
-        self._switch_to = MobileSwitchTo(self)
```

Comment:
do we use `_switch_to` property anywhere else?

Reply:
Thanks for your pointing out, I've checked it again, but it isn't used anywhere else.